packages/ckeditor5-dev-tests/tests/utils/manual-tests/compilescripts.js

```diff
@@ -13,41 +13,116 @@ const sinon = require( 'sinon' );
-		mockery.registerMock( 'webpack', webpackSpy );
-		sandbox.mock( path, 'join', ( ...chunks ) => chunks.join( '/' ) );
-
-		const compileManualTestScripts = require( '../../../lib/utils/manual-tests/compilescripts' );
-
-		return compileManualTestScripts( 'buildDir', 'manualTestPattern' ).then( () => {
-			expect( getWebpackConfigSpy.getCall( 0 ).args[ 0 ] ).to.deep.equal( { file1: 'file1', file2: 'file2' } );
-			expect( getWebpackConfigSpy.getCall( 0 ).args[ 1 ] ).to.deep.equal( 'buildDir' );
-			expect( webpackSpy.getCall( 0 ).args[ 0 ] ).to.deep.equal( { buildDir: 'buildDir', entries: { file1: 'file1', file2: 'file2' } } );
-		} );
+		stubs.glob.returns( [ 'file1', 'file2' ] );
+
+		return compileManualTestScripts( 'buildDir', [ 'manualTestPattern' ] )
+			.then( () => {
+				expect( stubs.getWebpackConfig.calledOnce ).to.equal( true );
+				expect( stubs.getWebpackConfig.firstCall.args[ 0 ] ).to.deep.equal( {
+					file1: 'file1',
+					file2: 'file2'
+				} );
+				expect( stubs.getWebpackConfig.firstCall.args[ 1 ] ).to.deep.equal( 'buildDir' );
+
+				expect( stubs.webpack.calledOnce ).to.equal( true );
+				expect( stubs.webpack.firstCall.args[ 0 ] ).to.deep.equal( {
+					buildDir: 'buildDir',
+					entries: {
+						file1: 'file1',
+						file2: 'file2'
+					}
+				} );
+
+				expect( stubs.glob.calledOnce ).to.equal( true );
+				expect( stubs.glob.firstCall.args[ 0 ] ).to.equal( 'manualTestPattern' );
+			} );
+	} );
+
+	it( 'resolves a few entry points patterns', () => {
+		const manualTestScriptsPatterns = [
+			'ckeditor5-build-classic/tests/**/manual/**/*.js',
+			'ckeditor5-editor-classic/tests/manual/**/*.js'
+		];
+
+		stubs.glob.onFirstCall().returns( [
+			'ckeditor5-build-classic/tests/manual/ckeditor.js',
+			'ckeditor5-build-classic/tests/manual/ckeditor.compcat.js'
+		] );
+
+		stubs.glob.onSecondCall().returns( [
+			'ckeditor5-editor-classic/tests/manual/classic.js'
+		] );
+
+		return compileManualTestScripts( 'buildDir', manualTestScriptsPatterns )
+			.then( () => {
+				expect( stubs.glob.calledTwice ).to.equal( true );
+				expect( stubs.glob.firstCall.args[ 0 ] ).to.equal( manualTestScriptsPatterns[ 0 ] );
+				expect( stubs.glob.secondCall.args[ 0 ] ).to.equal( manualTestScriptsPatterns[ 1 ] );
+
+				expect( stubs.getWebpackConfig.calledOnce ).to.equal( true );
+
+				expect( stubs.getRelativeFilePath.calledThrice ).to.equal( true );
+				expect( stubs.getRelativeFilePath.firstCall.args[ 0 ] )
+					.to.equal( 'ckeditor5-build-classic/tests/manual/ckeditor.js' );
+				expect( stubs.getRelativeFilePath.secondCall.args[ 0 ] )
+					.to.equal( 'ckeditor5-build-classic/tests/manual/ckeditor.compcat.js' );
+				expect( stubs.getRelativeFilePath.thirdCall.args[ 0 ] )
+					.to.equal( 'ckeditor5-editor-classic/tests/manual/classic.js'
+				);
+			} );
+	} );
+
+	it( 'rejects if Webpack threw an error', () => {
+		webpackError = new Error( 'Unexpected error.' );
```

Comment:
It cannot be - https://github.com/ckeditor/ckeditor5-dev/pull/138/files/68fa55db7e967752cd5730c8c6b8dd10e4038cef#diff-8e851cac5d1c9bb49f67678d7df1a613R30.

This variable is using as a parameter for callback in Webpack mock.